src/cpp/session/CMakeLists.txt

```diff
@@ -332,6 +334,7 @@ target_link_libraries(rsession
    ${SESSION_SYSTEM_LIBRARIES}
    ${LIBR_LIBRARIES}
    ${CMAKE_DL_LIBS}
+   ${Boost_LIBRARIES}
```

Comment:
You're right, it wasn't necessary (I think I erroneously added it in because I thought it was necessary to get `timer`; which is properly handled earlier. This was done to support some simple profiling macros)